data/json/items/armor/cloaks.json

```
@@ -457,6 +457,13 @@
         "weight": 10,
         "append": true
       },
+      {
+        "id": "car_flag",
+        "name": { "str": "Central African Republic flag" },
+        "description": "This is the flag of the Central African Republic, vertically striped with blue, white, green and yellow coloration and a one horizontal red stripe down the middle with a star in the upper left corner.",
```

```suggestion
        "id": "central_african_flag",
        "name": { "str": "Central African flag" },
        "description": "A flag with vertical stripes of blue, white, green and yellow coloration and a one horizontal red stripe down the middle with a star in the upper left corner.",
```
Just changes it to match the others using the adjectival rather than country name and removing the repetition